src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const arrValues = Object.values(robot);
+  const uniqueKeys = [...new Set(arrValues)];
+
+  if (arrValues.length !== uniqueKeys.length) {
+    return null;
+  }
+
+  for (const keys in robot) {
+    const valuesObj = robot[keys];
```

Comment:
Тут не буде значень об'єкта просто значення
```suggestion
    const part = robot[keys];
```